tools/schemacode/pyproject.toml

```diff
@@ -1,11 +1,65 @@
 [build-system]
-requires = ["setuptools"]
-build-backend = "setuptools.build_meta"
+build-backend = "hatchling.build"
+requires = ["hatchling"]
+
+[project]
+authors = [{name = "Nilearn developers"}]
+classifiers = [
+    "Development Status :: 4 - Beta",
+    "License :: OSI Approved :: MIT License",
+    "Intended Audience :: Science/Research",
+    "Topic :: Scientific/Engineering :: Information Analysis",
+    "Programming Language :: Python :: 3.9",
+    "Programming Language :: Python :: 3.10",
+    "Programming Language :: Python :: 3.11",
+    "Programming Language :: Python :: 3.12",
+    "Programming Language :: Python :: 3.13"
+]
+dependencies = [
+    "click",
+    "pyyaml",
+    "importlib_resources; python_version < '3.9'",
+    "jsonschema"
+]
+description = " Python tools for working with the BIDS schema."
+license = {text = "MIT"}
+maintainers = [{name = "bids-standard developers", email = " bids.maintenance@gmail.com"}]
+name = "bidsschematools"
+readme = "README.md"
+requires-python = ">=3.9"
+version = "0.11.4-dev"  # file:bidsschematools/data/schema/SCHEMA_VERSION
```

Comment:
hard coded for now until I find how to read version from file

Reply:
```suggestion
dynamic = ["version"]
```